packages/react/src/components/FlyoutMenu/FlyoutMenu.jsx

```diff
@@ -398,8 +400,8 @@ const propTypes = {
 
   /** Set of internationalized button names */
   i18n: PropTypes.shape({
-    cancelButton: PropTypes.string,
```

Comment:
Is this breaking change?

Reply:
No, it was never correct to begin with. https://github.com/carbon-design-system/carbon-addons-iot-react/blame/0019171e71520aa92357a3b059120747e58cca08/packages/react/src/components/FlyoutMenu/FlyoutMenu.jsx